src/bootman/bootman.c

```diff
@@ -63,6 +65,61 @@ BootManager *boot_manager_new()
         return r;
 }
 
+/**
+ * Set the command line to append into cmdline options on a global basis
+ */
+static void boot_manager_load_cmdline(BootManager *self)
+{
+        autofree(FILE) *f = NULL;
+        autofree(char) *cmdline = NULL;
+        size_t sn;
+        ssize_t r = 0;
+        char *buf = NULL;
+
+        if (self->cmdline) {
+                free(self->cmdline);
+                self->cmdline = NULL;
+        }
+
+        if (asprintf(&cmdline, "%s/etc/kernel/cmdline", self->sysconfig->prefix) < 0) {
```

Comment:
So you have the /etc/kernel `#defined` in another change, it seems like something you could take as a configure too potentially.